docs/installation.rst

```diff
@@ -111,4 +111,24 @@ from tardis/_compiler.c:1: /Users/yssavo/miniconda2/envs/tardis-show2/lib/gcc/x8
 
 **Problem:** Symbol not found: _GOMP_parallel when compiling with `--with-openmp`
 
-**Solution:** Install gcc8 from macports and then install with these flags: `link_args = ['-fopenmp','-Wl,-rpath,/opt/local/lib/gcc8/']`
\ No newline at end of file
+**Solution:** Install gcc8 from macports and then install with these flags: `link_args = ['-fopenmp','-Wl,-rpath,/opt/local/lib/gcc8/']`
+
+**Problem:** While building tardis(via python 2.7) via ``python setup.py`` build you
+may encounter the following error::
+
+     TypeError: super() argument 1 must be type, not None
+    
+    ----------------------------------------
+    Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-req-build-wPB39p/
+
+
+**Solution:** The cause for this problem is sphinx , or sphinx version . It can be easily solved by installing sphinx 1.5.6.
+              The command for the same is :
+
+    pip install sphinx==1.5.6
+    
+    or
+    
+    conda install sphinx==1.5.6
```

Comment:
I think it is just one `=` but try it out.

Reply:
I tried it out with a double ```=``` and it worked . This is what I got as the output :  
```
  added / updated specs: 
    - sphinx==1.5.6


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    sphinx-1.5.6               |           py27_0         1.3 MB
    anaconda-custom            |   py27h4a00acb_0           6 KB
    openssl-1.1.1b             |       h7b6447c_0         4.0 MB
    ------------------------------------------------------------
                                           Total:         5.3 MB
```
I think that works